deploy/kubernetes/setup-sumologic.yaml.tmpl

```diff
@@ -247,6 +351,12 @@ data:
       type  = string
       default = "$NAMESPACE"
     }
+  
+    variable "create_fields" {
+      description = "If set, terraform will attempt to create fields at Sumo Logic"
+      type        = bool
+      default     = false
```

Comment:
Why default to false?

Reply:
No specific reason. I've provided the default as suggested by @sumo-drosiek prevent the need for specifying terraform variable during imports.

We can also default to `true` if that's more appropriate for our use case. It will be specified on `terraform apply` anyway

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/bc18a3e2bfe0285484a777a561a38946325b8e34/deploy/helm/sumologic/conf/setup/setup.sh#L79-L81

Reply:
After discussion with folks on PL sync I've changed this to `true` (which doesn't really change anything in our code but expresses our intent of creating fields by default)